### DISCUSSION TITLE
Welcome to TorBot Discussions!

### DISCUSSION THREAD

**Discussion by PSNAppz** [DedSecInside/TorBoT#197] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by cackymonger** [comment]
Is the -list flag no longer a thing? I'm getting an error saying it is an unrecognized argument.

**Reply by KingAkeem** [reply]
I don't remember this flag ever being supported, it's not listed under the official flags. Check the README

**Reply by cackymonger** [reply]
Sorry. I meant -live

**Reply by KingAkeem** [reply]
No problem and the `-live` flag does appear to be removed since the default behavior of torBot with the `-u` flag is the same so using both would be redundant. It doesn't appear that the documentation has been updated to reflect this but if you run torBot with the `-h` flag then you'll no longer see the `-live` parameter.

**Reply by KingAkeem** [reply]
Here's a PR to correct the documentation: https://github.com/DedSecInside/TorBot/pull/203, thank you for pointing that out.

**Comment by luke2103** [comment]
> (venv) lukesh@pop-os:~/TorBot/src$ python3 torBot.py -u http://duskgytldkxiuqc6.onion/ -d

                                    
                           __  ____  ____  __        ______
                          / /_/ __ \/ __ \/ /_  ____/_  __/
                         / __/ / / / /_/ / __ \/ __ \/ /
                        / /_/ /_/ / _, _/ /_/ / /_/ / /
                        \__/\____/_/ |_/_____/\____/_/  V1.4.0
              
                    #######################################################
                    #  TorBot - An OSINT Tool for Dark Web                #
                    #  GitHub : https://github.com/DedsecInside/TorBot    #
                    #  Help : use -h for help text                        #
                    #######################################################
                                  LICENSE: GNU Public License v3
              

> Attempting to connect to https://check.torproject.org/
Tor IP Address: 185.220.101.135
Adding node for:  "Woooooooooooo!" -Roger"
Traceback (most recent call last):
  File "torBot.py", line 250, in <module>
    main()
  File "torBot.py", line 169, in main
    tree = LinkTree(node)
  File "/home/lukesh/TorBot/src/modules/analyzer.py", line 33, in __init__
    self._tree = build_tree(root_node, stop=stop_depth)
  File "/home/lukesh/TorBot/src/modules/analyzer.py", line 90, in build_tree
    if child.get_children():
  File "/home/lukesh/TorBot/src/modules/link.py", line 122, in get_children
    return self._children
AttributeError: 'LinkNode' object has no attribute '_children'






Can someone help me with this . Why I'm getting this erro and I'm getting the error when using -v instead of d 

Thank you in advance for helping

**Comment by pulkit22** [comment]
Getting problem with the installation.

Could somebody please give a step-by-step process for proper installation?

**Reply by cackymonger** [reply]
Here's a step by step I did on a fresh Kali install....its almost a year old though so no guarantees it will help.

http://cackymonger.com/index.php/2021/03/31/installing-torbot-maybe-part-1-of-a-bigger-project/

**Reply by pulkit22** [reply]
Thank you

**Reply by pulkit22** [reply]
![TorBot](https://user-images.githubusercontent.com/39116298/153835138-58df20cc-3fcc-4601-b85f-09e07eb1a1b9.png)
Getting this error  when trying to execute totBot.py

**Reply by pillocx** [reply]
> Getting problem with the installation.
> 
> Could somebody please give a step-by-step process for proper installation?

I have same issues, could somebody help me, thanks

**Reply by PSNAppz** [reply]
> Here's a step by step I did on a fresh Kali install....its almost a year old though so no guarantees it will help.
> 
> http://cackymonger.com/index.php/2021/03/31/installing-torbot-maybe-part-1-of-a-bigger-project/

That's crazy. Hopefully, I have released a new version that fixes most of the issues with the ete3 library (Spoiler: I had to remove the library from torbot). But since the visualization module was the least used one, I guess it's okay.
